### ISSUE TITLE
doubt about hessians calculation 

### ISSUE THREAD

**Issue by hlee13** [Microsoft/LightGBM#1154]
https://github.com/Microsoft/LightGBM/blob/master/src/objective/multiclass_objective.hpp:73,94

Logistic regression is a special case of softmax:

I have no doubt about gradients is (p - label), but hessians is 2.0f * p * (1.0f - p), why not p * (1.0f - p)

**Comment by guolinke** [comment]
I remember there are some orher issues related to this, you can seach for it.

**Comment by wxchan** [comment]
https://github.com/Microsoft/LightGBM/issues/1052

**Comment by hlee13** [comment]
thanks.
